var/spack/repos/builtin/packages/hermes-shm/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class HermesShm(CMakePackage):
+    """Hermes Shared Memory contains a variety of data structures
+    and synchronization primitives which are compatible with shared memory.
+    """
+
+    homepage = "https://github.com/grc-iit/hermes-shm/wiki"
+    git = "https://github.com/grc-iit/hermes-shm.git"
+
+    maintainers("lukemartinlogan", "hyoklee")
+
+    version("master", branch="master")
+    version(
+        "1.1.0",
+        url="https://github.com/grc-iit/hermes-shm/archive/refs/tags/v1.1.0.tar.gz",
+        sha256="2270d629373447a2872d7109b5a5e66027dc8d8178d3ba84eb48a875f49b6bdf",
+    )
+    version(
+        "1.0.0",
+        url="https://github.com/lukemartinlogan/hermes_shm/archive/refs/tags/v1.0.0.tar.gz",
+        sha256="a79f01d531ce89985ad59a2f62b41d74c2385e48d929e2f4ad895ae34137573b",
+    )
+
+    # Main variants
+    variant("mochi", default=True, description="Build with mochi-thallium support")
+    variant("cereal", default=True, description="Build with cereal support")
+    variant("boost", default=True, description="Build with boost support")
+    variant("mpiio", default=True, description="Build with MPI support")
+    variant("vfd", default=False, description="Build with HDF5 support")
+    variant("zmq", default=False, description="Build ZeroMQ tests")
+    variant("adios", default=False, description="Build Adios support")
+
+    # Required deps
+    depends_on("catch2@3.0.1")
+    depends_on("yaml-cpp")
+    depends_on("doxygen@1.9.3")
```

Comment:
Is this more appropriate:
```suggestion
    depends_on("doxygen@1.9.3", type="build")
```
?